asn1ate/sema.py

```diff
@@ -271,14 +271,23 @@ def user_types(self):
 
         return self._user_types
 
-    def resolve_type_decl(self, type_decl):
+    def resolve_type_decl(self, type_decl, referenced_modules = None):
```

Comment:
`referenced_modules` will always be provided, and never be `None`, right? If you can make that assumption true, you can simplify the implementation a little.